New/exp2

Corvax357 opened the pull request.

Corvax357 committed:
Create HTML4FUN

Corvax357 committed:
Delete README.md

Corvax357 committed:
Delete okDoc

Corvax357 committed:
Delete okDoc-2

Corvax357 committed:
Delete HTML4FUN

Corvax357 committed:
Create README.md

Corvax357 committed:
Create index.html

Corvax357 committed:
Create styles.css

Corvax357 committed:
Layout

Some layout for html-doc.

Corvax357 committed:
Create .gitignore

ostaptan commented:
Try to minimize count of commits. Commits should be spliced logically. In this case you should have had 3 commits:
1) Create basic staff
2) Delete old staff
3) Add layout

ostaptan commented on css/styles.css:
```diff
@@ -0,0 +1,65 @@
+
+body {background-color:#FFFFFF; 
```
use css standard coding conventions and best practices to organize code.
http://www.phpied.com/css-coding-conventions/

ostaptan commented on html/index.html:
```diff
@@ -0,0 +1,261 @@
+
+<table id="t1">
+	<tr>
+		<td colspan=5 rowspan=3>
```
use 2 spaces for identation

ostaptan commented on css/styles.css:
```diff
@@ -0,0 +1,65 @@
+
+#t2, #t3, #t4 {border: 0px; border-collapse:collapse; bordercolor: #111111; width: 100%;}  
+
+#l1 {width: 364px; height: 28px}
```
### this is awful!!!

1) do not use ids. ids are used for javascript **ONLY**!!! use classes.
2) use class nesting, use global classes to avoid duplication and mess.
3) use descriptive class names so it is easily to be found and understandable.
4) HELP: use multiple classes so your code is more elegant
Example:

```
<td class='cool fantastic small elegant magical karma satori smile'>
```

ostaptan commented on css/styles.css:
```diff
@@ -0,0 +1,65 @@
+#a1, #a2, #a3, #a4 {font-size: 14px; color:black}
+
+
+:active     {  font-family: Arial; style; font-size: 12px; color: #FFFFFF; letter-spacing:0px}
```
Maximally avoid duplication!

ostaptan commented on html/index.html:
```diff
@@ -0,0 +1,261 @@
+                <img id="l31" src="images/layout_31.gif"></td>
+              </tr>
+            </table>
+			<img id="l32" src="images/layout_32.gif" ></td>
```
http://joxi.ru/Q2K0RDnse8B6Aj

Have you even tested you html? 
your images are stored in "vendor" folder. But your src is pointing to "images" folder.

ostaptan commented:
**WARNING**!!! Please do not **COPY-PASTE** code from internet! This increase amount of bugs!

ostaptan commented on html/index.html:
```diff
@@ -0,0 +1,261 @@
+			<img id="l4" src="images/layout_04.gif" alt=""></td>
+	</tr>
+	<tr>
+		<td colspan=8 background="images/layout_05.gif" height="18">
```
all styles **should** be only in css file!!! 
It is **forbidden** to mess css code with html code!

ostaptan commented on html/index.html:
```diff
@@ -0,0 +1,261 @@
+		<td colspan=8 background="images/layout_05.gif" height="18">
+		
+                <b> 
+            <a id="a1" href="index.htm"> HOME </a>  *
```
### Please be more patient

It looks like your file name is _index.html_ NOT _index.htm_

Corvax357 committed:
Corrected html-doc

Short summary of changes:
1. All styles were moved to CSS-file (please treat images in html-elements as a content).
2. IDs were replaced by classes/pseudo-classes.
3. Useless tags were removed/improved.
4. Layout positioning was improved (no "white stripes" left).

Corvax357 committed:
Corrected css-doc

CSS were styled due to "coding conventions".

P.S.
Please notice that the "bad" html-doc structure were not changed - so many duplicating styling elements could be omitted (for example: .spacer).
Also there were no necessity to use nested/global classes due to html-doc structure.

Corvax357 committed:
Create LICENSE.md

ostaptan commented on css/styles.css:
```diff
@@ -0,0 +1,134 @@
+}
+
+table {
+    border: 0px;
```
Use 2 spaces for indentation.

Corvax357 committed:
Corrected css-doc

Indentation changed from 4 spaces to 2.

Corvax357 committed:
Update index.html

Layout filepath fixed.
